experiment/run_experiment.py

```diff
@@ -66,12 +66,27 @@ def read_and_validate_experiment_config(config_filename: str) -> Dict:
     # it isn't like anything expensive happens before this validation is done so
     # rerunning it is cheap.
     config = yaml_utils.read(config_filename)
-    bucket_params = {'cloud_experiment_bucket', 'cloud_web_bucket'}
-    string_params = {
-        'cloud_compute_zone', 'cloud_experiment_bucket', 'cloud_web_bucket'
-    }
+    cloud_bucket_params = {'cloud_experiment_bucket', 'cloud_web_bucket'}
```

Comment:
Why are these still called bucket? I think we shouldn't support `cloud_{experiment,web}_bucket` anymore, users will specify a `{experiment,cloud}_filestore` that will be used for local and cloud experiments.